src/redturtle/prenotazioni/content/prenotazioni_folder.py

```diff
@@ -209,6 +209,26 @@ def notify_on_refuse_message_default_factory(context):
     )
 
 
+@provider(IContextAwareDefaultFactory)
+def notify_as_reminder_subject_default_factory(context):
+    return getattr(context, "translate", translate)(
+        _(
+            "notify_as_reminder_subject_default_value",
+            "You have the upcomming booking on ${booking_date}",
+        )
+    )
+
+
+@provider(IContextAwareDefaultFactory)
+def notify_as_reminder_message_default_factory(context):
+    return getattr(context, "translate", translate)(
+        _(
+            "notify_as_reminder_message_default_value",
+            'Booking details are available by the following <a href="${booking_pring_url}">link</a>',
```

Comment:
I will write down more infos about the booking.
Something like:

This is an automatic reminder about your booking on ${date} for ${booking_type}.

If you need to see more infos or delete it, please access your <a>booking details page</a>.

Reply:
Solved

Reply:
Solved